ui/perfherder/alerts/AlertsViewControls.jsx

```diff
@@ -107,7 +109,7 @@ export default class AlertsViewControls extends React.Component {
           dropdownCol
         />
         {pageNums
-          ? pageNums.length > 1 && (
+          ? hasMorePages > 1 && (
```

Comment:
I think `? hasMorePages && (` is the correct version, because `hasMorePages` returns a bool value.